### PR TITLE
Adding landing page of notes

### PR DESCRIPTION
**Pull request by bipin7yadav**
- login page
- signUp pag
notes-landing-page
e

### CODE REVIEW

File: src/components/navigation/navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                </div>
+            </div>
+
+            <div className="cartWish">
+                <h2 className="icon"><i class="material-icons ">login</i> </h2>
+                {/* <i class="material-icons ">login</i> */}
+                <h2 className="icon">
+                    <button className="sortBy">
+                        {/* <img className="sort" src="https://cdn-icons-png.flaticon.com/128/318/318146.png" /> */}
+                        <i class="material-icons">sort</i>
```
Comment (gopalbbharadva) [inline src/components/navigation/navigation.jsx:25]: Remove unused code.

File: src/pages/landing-page/landingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <>
+            <div className="container">
+                <div>
+                    <img className="note-img" src="./assets/notesApp.svg" />
+                </div>
+
+                <div className="links">
+                    <Link to="./home">
+                        <button className="Cbtn">CREATE NOTE</button>
+                    </Link>
```
Comment (gopalbbharadva) [inline src/pages/landing-page/landingPage.jsx:14]: Suggestion: You should give bgcolor to  primary color of  CTA button

**Review by gopalbbharadva — APPROVED** [review]
Good to go.

**Review by preethasri — APPROVED** [review]
good work

**Pull request merged**